crit/Makefile

```diff
@@ -0,0 +1,26 @@
+GO ?= go
+
+# The import path that protoc will use if a proto file imports another one
+import_path := github.com/checkpoint-restore/go-criu/crit/images
+# Path to .proto source files
+proto_path := ./images
+# Generate string of all .proto filenames
+proto_files := $(sort $(subst $(proto_path)/,,$(wildcard $(proto_path)/*.proto)))
+# Generate M flag to specify import path for all .proto files
+# and replace all spaces with commas to use with go_opt flag
+comma := ,
+proto_opts := $(subst $() $(),$(comma),$(patsubst %,M%=$(import_path),$(proto_files)))
+
+get-proto:
+	git clone https://github.com/checkpoint-restore/criu criu-temp
```

Comment:
You can add `--depth=1` and `-b master` to download only the latest commit from the master branch.